imap_processing/tests/hi/test_hi_l1b.py

```diff
@@ -152,6 +157,36 @@ def test_compute_coincidence_type_and_time_deltas(synthetic_trigger_id_and_tof_d
     )
 
 
+@mock.patch("imap_processing.hi.l1b.hi_l1b.parse_sensor_number", return_value=90)
+@mock.patch("imap_processing.hi.l1b.hi_l1b.get_instrument_spin_phase")
+def test_compute_instrument_spin_phase(parse_sensor_number_mock, instrument_phase_mock):
+    """Test coverage for compute_instrument_spin_phase."""
+    # set the get_instrument_spin_phase mock to return an array of values between
+    # 0 and 1
+    parse_sensor_number_mock.side_effect = lambda x, y: np.linspace(0, 1, len(x))
```

Comment:
Learn something new about `.side_effect`!

But this questions is a curiosity question. How is `x, y` used here? I thought `parse_sensor_number()` takes in only one argument. I see that you use this returned value to compare at `line 185`.

Reply:
Haha, I have my mock names mixed up. The argument order from decorators is reversed from the decorator order. This is actually setting the side_effect for the thing mocking `get_instrument_spin_phase` which is what I intended. And that function takes two arguments. I'll fix that.